generators/app/tasks/createBitriseApp.js

```diff
@@ -4,90 +4,84 @@ const gitlabApi = require('../apis/gitlabApiConfig');
 const bitbucketApi = require('../apis/bitbucketApiConfig');
 
 const gitApi = {
-    github: githubApi,
-    gitlab: gitlabApi,
-    bitbucket: bitbucketApi
+  github: githubApi,
+  gitlab: gitlabApi,
+  bitbucket: bitbucketApi
 };
 
-const createApp = async ({repoUrl, isPublic, gitOwner, provider, repoSlug, type}) => {
-    return await bitriseApi.post('/apps/register', {
-        git_owner: gitOwner,
-        is_public: isPublic,
-        provider: provider,
-        repo_url: repoUrl,
-        git_repo_slug: repoSlug,
-        type: type
-      })
-};
+const createApp = ({ repoUrl, isPublic, gitOwner, provider, repoSlug, type }) =>
+  bitriseApi.post('/apps/register', {
+    git_owner: gitOwner,
+    is_public: isPublic,
+    provider,
+    repo_url: repoUrl,
+    git_repo_slug: repoSlug,
+    type,
+    organization_slug: '91af5dfae6f01368'
```

Comment:
Be careful, this repo is public and is thought for use not only Wolox. All Bitrise data will be configurable via params or env vars.